src/test/locale/fr.js

```diff
@@ -3,11 +3,13 @@ import moment from '../../moment';
 localeModule('fr');
 
 test('parse', function (assert) {
-    var tests = 'janvier janv._février févr._mars mars_avril avr._mai mai_juin juin_juillet juil._août août_septembre sept._octobre oct._novembre nov._décembre déc.'.split('_'),
-        i;
+    var i,
```

Comment:
There is indeed no functional change in changing the variables order. It just happens to be in this reversed order inside the fr-ch and fr-ca files, and I'm trying to align all fr sub-locales files.

I explained it in the PR description. This change is added only so that the 3 fr locales test files share as much code as possible, so that a diff between any 2 files of fr-ca, fr-ch or fr show only actual differences between the fr sub-locales.